stor/s3.py

```diff
@@ -468,6 +468,11 @@ def stat(self):
         }
         return response
 
+    @property
+    def content_type(self):
+        """Get content type for path (using ContentType field from Boto) or empty string."""
+        return self.stat().get('ContentType', '') or ''
```

Comment:
nit: is `or ''` needed? doesn't `.get('ContentType', '')` already handle that?

Reply:
sure changed it to how you had it